plugins/woocommerce-blocks/assets/js/blocks/collection-filters/edit.tsx

```diff
@@ -23,13 +29,15 @@ if ( firstAttribute ) {
 	] );
 }
 
-const Edit = () => {
+const Edit = ( { clientId }: BlockEditProps< Record< string, never > > ) => {
```

Comment:
`never`? 

that can't be right?

Either we should type this or it should be `unknown`?

Reply:
Yeah, we should type it. I fixed it by reusing the EditProps you added in other PR.